p2p/peer.go

```diff
@@ -490,5 +493,18 @@ func (p *Peer) Info() *PeerInfo {
 		}
 		info.Protocols[proto.Name] = protoInfo
 	}
+
+	var staticNodeLabels []string
+	for label := range p.StaticNodeLabels {
+		staticNodeLabels = append(staticNodeLabels, label)
+	}
+	info.StaticNodeLabels = staticNodeLabels
+
+	var trustedNodeLabels []string
+	for label := range p.TrustedNodeLabels {
+		trustedNodeLabels = append(trustedNodeLabels, label)
+	}
+	info.TrustedNodeLabels = trustedNodeLabels
+
```

Comment:
One thing I ran into when making the sentry infrastructure is the inability to use the sentry's geth console to print peers on its internal/proxy interface. I had to use `admin.peers` on the proxied validator. I think it'd be helpful to either have a separate command like `admin.proxiedPeers`

Reply:
I agree that this is helpful.  Would it be very helpful for you in the short term?  If not, are you ok with making this a betanet+1 issue?

Reply:
I was able to get things working the way it is so there's no rush on my side. betanet+1 works for me

Reply:
Created a ticket:  https://github.com/celo-org/celo-blockchain/issues/576